cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/RebalanceOptions.java

```diff
@@ -130,6 +139,11 @@ public RebalanceOptionsBuilder withSkipHardGoalCheck() {
             return this;
         }
 
+        public RebalanceOptionsBuilder withRebalanceDisk(boolean rebalanceDisk) {
```

Comment:
Why does this need the boolean argument? The default is false, if someone calls rebalanceBuilder.withRebalanceDisk() we set the value to true. If not it stays false?